src/partials/support-status-banner/banner-for-eol-scheduled.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="notice-banner">
```

Comment:
@hannanelson you can look at the wording of all the banner-for-* files starting from this one.